### PR TITLE
Allow thumbnail and asset dimensions to be queried

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
not sure why the tests fail...

File: src/GraphQL/Resolver/AssetType.php [hunk 1]
```diff
@@ -250,6 +250,48 @@ public function resolveResolutions($value = null, $args = [], $context = [], Res
...
+     *
+     * @throws \Exception
+     */
+    public function resolveDimensions($value = null, $args = [], $context = [], ResolveInfo $resolveInfo = null)
+    {
+        if ($value instanceof ElementDescriptor) {
+            $thumbnailName = $args['thumbnail'] ?? null;
+
+            /**
+             * @var Asset\Image $asset
```
Comment (weisswurstkanone) [inline src/GraphQL/Resolver/AssetType.php:269]: Is it always of type image ?

Comment (dpfaffenbauer) [reply src/GraphQL/Resolver/AssetType.php:269]: it is for resolveResolution too.

**Comment by weisswurstkanone** [comment]
What I wanted to ask ... for a text asset, what would be the result ? :-) Otherwise it looks good to me

**Comment by dpfaffenbauer** [comment]
To be honest, I did it the same as `resolutions`, what happens there if it as text asset?

**Comment by weisswurstkanone** [comment]
Hi @dpfaffenbauer , I would assume that it will crash :-) Can you give it a try please ?

**Comment by dpfaffenbauer** [comment]
Yes, it does. Resolutions crashes as well as dimensions:

Bildschirmfoto 2021-07-14 um 13 12 09

**Comment by weisswurstkanone** [comment]
See attached PR #388 which should fix this.

Works for 
{
  getAsset(id: 68) {
    filename
    resolutions(thumbnail: "exampleColor") {      
      url
    }
  }
}


but not for id 336 because 336 is not an image/video asset.

**Comment by dpfaffenbauer** [comment]
changed it on my side

**Comment by weisswurstkanone** [comment]
great, thanks!

**Pull request merged**